.github/workflows/rust.yml

```
@@ -37,8 +40,36 @@ jobs:
+      - name: Install Rust ${{ env.MSRV }}
+        run: rustup install ${{ env.MSRV }} --profile minimal
+      - name: Verify MSRV version
+        run: cargo +${{ env.MSRV }} -V
+      - name: Build with MSRV
+        run: cargo +${{ env.MSRV }} build
+      - name: Build with all features
+        run: cargo +${{ env.MSRV }} build --all-features
+
   test:
-    needs: lint
+    needs: [ lint, msrv ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)